lib/wallet.js

```diff
@@ -68,75 +77,79 @@ class Wallet extends ethers.Signer {
+
+        const address = await this.getAddress();
+        dbg('Sending ' + amountWei + ' wei to ' + clientFund.address + ' from ' + address);
         return this.sendTransaction(rawTx);
     }
-    
+
     /**
-     * Initiates the deposit of a token from the wallet's on-chain balance to nahmii by calling the 
-     * approve method of the token smart contract.
+     * Initiates the deposit of a token from the wallet's on-chain balance to
+     * nahmii by calling the approve method of the token smart contract.
      * @param {(number|string)} amount - The amount of currency to deposit.
      * @param {string} symbol - The currency symbol
      * @param [options]
-     * @returns {Promise} A promise that resolves into a transaction with a hash.
+     * @return {Promise} A promise that resolves into a transaction with a hash.
      * @see https://docs.ethers.io/ethers.js/html/api-providers.html#transaction-receipts
      * @example
      * const {hash} = await wallet.depositToken('1.1', 'HBT', {gasLimit: 200000});
      * const receipt = await wallet.provider.getTransactionConfirmation(hash);
      */
     async approveTokenDeposit(amount, symbol, options) {
-        options = { gasLimit: 600000, ...options };
+        options = {gasLimit: 600000, ...options};
         const tokenInfo = await getTokenInfo.call(this, symbol);
         const tokenContract = new Erc20Contract(tokenInfo.currency, this);
         const amountBN = ethers.utils.parseUnits(amount.toString(), tokenInfo.decimals);
         const clientFund = _clientFund.get(this);
 
         try {
             return await tokenContract.approve(clientFund.address, amountBN, options);
-        } catch(e) {
+        } catch (e) {
             throw new Error(`Failed to approve token deposit: ${e}`);
         }
     }
 
     /**
-     * Initiates the completion of a deposit of a token from a wallet's on-chain balance 
-     * to nahmii by calling the depositTokens method of the nahmii clientFund smart contract.
+     * Initiates the completion of a deposit of a token from a wallet's on-chain
+     * balance to nahmii by calling the depositTokens method of the nahmii
+     * clientFund smart contract.
      * Requires approveTokenDeposit to have been called first.
      * @param {(number|string)} amount - The amount of currency to deposit.
      * @param {string} symbol - The currency symbol
      * @param [options]
-     * @returns {Promise} A promise that resolves into a transaction with a hash.
+     * @return {Promise} A promise that resolves into a transaction with a hash.
      * @example
      * const {hash} = await wallet.completeTokenDepsoit('1.1', 'HBT', {gasLimit: 200000});
      * const receipt = await wallet.provider.getTransactionConfirmation(hash);
      */
     async completeTokenDeposit(amount, symbol, options) {
-        options = { gasLimit: 600000, ...options };
+        options = {gasLimit: 600000, ...options};
```

Comment:
This is repeated a few times. Would it make sense to have it defined as a constant somewhere?